contracts/sfc/SFCState.sol

```diff
@@ -106,4 +106,13 @@ contract SFCState is Initializable, Ownable {
     ConstantsManager internal c;
 
     address public voteBookAddress;
+
+    struct Sponsor {
+        uint256 gasLimit;
+        uint256 index;
+    }
+    //nominee -> sponsors list
+    mapping(address => address[]) public sponsors;
```

Comment:
is there any bound for the array?

Reply:
The Sponsor array is unbounded. To save a gas and avoid reaching block's gas limit we only loop through the whole array inside view functions. All state changing operations use indexes to modify the array.